cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +61,22 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: bprashanth/healthz-server:0.0 # TODO: Move to gcr.io
+        resources:
+          limits:
+            cpu: 100m
+            memory: 50Mi
```

Comment:
Just took whatever we're using for the others. probably a lot smaller, will check.